app/Metadata/Extractor.php

```diff
@@ -10,6 +10,15 @@
 
 class Extractor
 {
+	/*
+	 * https://github.com/LycheeOrg/Lychee/issues/1200
```

Comment:
Please remove that first line. The first line is used by PHPdoc to generate a summary and by IDEs to generate a tooltip. A reference to an issue which will (hopefully) be closed, is not helpful.

Reply:
I can see an argument for moving it to a lower position, however it is very common to annotate a given bit of logic with a link to the issue it stemmed from, but it provides an easy way to gain some additional context as to why something has been implemented as it has without having to populate the entire context in-code.

Assuming such context is desired.  If not, I can remove it.

Reply:
I don't argue against context, but the context should be in-place.